test/language/statements/for/S12.6.3_A10.1_T2.js

```diff
@@ -0,0 +1,113 @@
+// Copyright 2009 the Sputnik authors.  All rights reserved.
```

Comment:
S12.6.3_A10.1_T2 is a copy of S12.6.3_A10.1_T1, therefore I've retained the original copyright notice.